app/auth/sign-up/page.tsx

```diff
@@ -70,41 +83,27 @@ const SignUp = () => {
       return;
     }
 
-    // TODO: handle role better - probably need a different route for users vs admins
+    // Generate payload for sign up
     const payload = {
       name: firstName + " " + lastName,
```

Comment:
name should be separated into firstName, lastName
<img width="189" alt="image" src="https://github.com/SeattleColleges/nsc-events-nextjs/assets/72054441/74ea5201-1ff7-491f-b5cd-67da52125309">

Reply:
![image](https://github.com/SeattleColleges/nsc-events-nextjs/assets/35015441/cb9f52ca-b3a2-4837-943f-11e49915cc14)

Currently the backend uses a single name field. Refactoring the backend is beyond the scope of this PR

Reply:
> Currently the backend uses a single name field. Refactoring the backend is beyond the scope of this PR
The backend was refactored 2 weeks ago.

Reply:
I have changed the fields to reflect that change.